docker-compose.yaml.template

```diff
@@ -14,6 +16,8 @@ services:
         - "prometheus-job=save-orchestrator"
   backend:
     image: 127.0.0.1:6000/save-backend:{{project.version}}
+    environment:
+      - SPRING_PROFILES_ACTIVE=prod
```

Comment:
Why is backend in prod only?

Reply:
It was a mistake. Environment now only in backend